app/src/main/java/com/example/composetemplate/presentation/common/LoginTextField.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.composetemplate.presentation.common
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.example.composetemplate.presentation.screens.entry_screens.login.LoginScreenEnum
+
+@Composable
+fun LoginTextField(
```

Comment:
It's looks like a complex component, but maybe here we can also make it more generic?

Reply:
RAZ same concept as LoginScreenButton